packages/core/src/crawlers/statistics.ts

```diff
@@ -300,18 +298,15 @@ export class Statistics {
     /**
      * Persist internal state to the key value store
      */
-    async persistState() {
-        // this might be called before startCapturing was called without using await, should not crash
-        if (!this.enablePersistence || !this.keyValueStore) {
+    async persistState(opts?: {
```

Comment:
same here, you can use the same interface (try to find some generic name that would fit both methods)